src/main/java/uk/gov/hmcts/darts/authorisation/model/GetAuthorisationResult.java

```diff
@@ -4,6 +4,7 @@ public record GetAuthorisationResult(Integer userId, String userName,
                                      Boolean globalAccess,
                                      Integer courthouseId,
                                      Integer roleId, String roleName,
-                                     Integer permissionId, String permissionName) {
+                                     Integer permissionId, String permissionName,
+                                     boolean active) {
```

Comment:
as a matter of consistency with the existing boolean field perhaps better define this as:
`Boolean active`?